classes/user/maps/Schema.php

```diff
@@ -165,6 +166,9 @@ protected function mapByProperties(array $props, User $user, array $auxiliaryDat
                                 'permitSelfRegistration' => (bool) $userGroup->getPermitSelfRegistration(),
                                 'permitMetadataEdit' => (bool) $userGroup->getPermitMetadataEdit(),
                                 'recommendOnly' => (bool) $userGroup->getRecommendOnly(),
+                                'dateStart' => UserUserGroup::withUserId($user->getId())
+                                   ->withUserGroupId($userGroup->getId())
+                                   ->pluck('date_start')->first()
```

Comment:
This could give you the start date of an old user_user_group, that already ended.
So you would need to also use `->withActive()`.
The API currently only considers the active user groups:
The line above: 
`$userGroups = Repo::userGroup()->userUserGroups($user->getId(), $this->context->getId());`
returns only active user groups.

Reply:
@bozana  I have changed the PR to get only the active users